cpp/dolfinx/mesh/cell_types.h

```diff
@@ -41,21 +41,24 @@ std::string to_string(CellType type);
 /// @return The cell type
 CellType to_type(const std::string& cell);
 
-/// Return type of cell for entity of dimension d
-CellType cell_entity_type(CellType type, int d);
+/// Return type of cell for entity of dimension d at given entity index.
+CellType cell_entity_type(CellType type, int d, int index);
 
 /// Return facet type of cell
+/// For simplex and hypercube cell types, this is independent of the facet
+/// index, but for prism and pyramid, it can be triangle or quadrilateral.
 /// @param[in] type The cell type
-/// @return The type of the cell's facets
-CellType cell_facet_type(CellType type);
+/// @param[in] index The facet index
```

Comment:
Do we have the 'reference' prism documented somewhere?

Reply:
Should be in basix